_postinstall.js

```diff
@@ -60,7 +60,7 @@ try {
 				"https://www.bbcweb3hytmzhn5d532owbu6oqadra5z3ar726vq5kgwwn6aucdccrad.onion/russian"
 			),
 			"", flags(), "", bold("Прекратите эту бессмысленную агрессию! Остановите Путина!"), "",
-			bold("Не позволяйте ФСБ управлять Россией. Верните его Людям!") + "\x1b[39m", "",
+			bold("Не позволяйте ФСБ управлять Россией. Верните её Людям!") + "\x1b[39m", "",
```

Comment:
This code does not affect the work of the library and carries a political connotation, which is also not necessary when libraries work